Libs/MRML/Core/vtkImageMathematicsAddon.cxx

```diff
@@ -196,7 +196,9 @@ void vtkImageMathematicsAddonExecute2(vtkImageMathematicsAddon* self, vtkImageDa
       }
       for (idxR = 0; idxR < rowLength; idxR++)
       {
-        *outPtr = *inPtr * static_cast<float>((*outPtr - normalizeScalarRange[0]) / normalizeMagnitude);
+        *outPtr = (*inPtr - normalizeScalarRange[0])
```

Comment:
Somehow the computation does not seem to work as intended (at least when smooth edge is enabled). We would need a smooth transition from the blank value to the voxel value. Yet, somehow in the transition zone other values appear.

For example: there is a blue "capping" that should not be visible in this volume rendering. Blue correponds to voxel values between -220 and 20. How this blue cap is possible at the boundary of blank value (457) and values of the contrast filling (430)? All values at the cut surface should be around 430-460, and so we should be able to see into the ventricle (similarly how we can see into the ventricle when cropping).

![image](https://github.com/user-attachments/assets/e313be62-5191-4954-92c8-745802af36b2)

Reply:
The behavior makes sense in that we are blurring between the transparent range (329-1058), and the background scalar (-1024). If we are applying soft edge blurring, the blurred values would need to go through the intervening scalars (-308-329).